submissions/israelOladosu/database/transportation.sql

```diff
@@ -0,0 +1,84 @@
+CREATE DATABASE transporation;
+
+use transporation;
+
+-- classes table
+CREATE TABLE classes (
+    id INT NOT NULL AUTO_INCREMENT,
+    class_level INT NOT NULL,
+    PRIMARY KEY(id)
+);
+
+-- passangers table
+CREATE TABLE passengers (
+    id INT NOT NULL AUTO_INCREMENT,
+    firstname VARCHAR(50) NOT NULL,
+    lastname VARCHAR(50) NOT NULL,
+    sex VARCHAR(6) NOT NULL,
+    ticket_number VARCHAR(20) NOT NULL,
+    age INT,
+    class_id INT NOT NULL,
+    PRIMARY KEY(id),
+    FOREIGN KEY(class_id) REFERENCES classes(id) ON DELETE CASCADE 
+);
+
+CREATE TABLE cabin_classes(
+    id INT NOT NULL AUTO_INCREMENT,
+    cabin_code VARCHAR(10) NOT NULL,
+    passenger_id INT NOT NULL,
+    INDEX(id),
+    PRIMARY KEY(passenger_id, cabin),
+    FOREIGN KEY(passenger_id) REFERENCES passengers(id) ON DELETE CASCADE
+);
+
+-- trips table
+CREATE TABLE trips (
+    id INT NOT NULL AUTO_INCREMENT,
+    passenger_class INT NOT NULL,
+    passenger_ticket VARCHAR(20) NOT NULL,
+    trip_fare DECIMAL(19,4) NOT NULL,
+    number_of_parents_children INT NOT NULL DEFAULT  0,
+    number_of_siblings_spouse INT NOT NULL DEFAULT 0,
+    assigned_cabin VARCHAR(40),
```

Comment:
This doesn't have to be here

Reply:
Not sure which one you are referring to as this... would appreciate if you can be more specific please